bin/tx-prover/README.md

```diff
@@ -61,6 +61,8 @@ max_retries_per_request = 1
 max_req_per_sec = 5
 # Interval to check the health of the workers
 health_check_interval_secs = 1
+# Port of the metrics server
+prometheus_port = 6192
```

Comment:
Not from this PR but we are missing the `available_workers_polling_time_ms` in this example config

Reply:
I added it in this PR, it was simple enough.